internal/cloud/aws.go

```diff
@@ -54,10 +54,39 @@ type Placement struct {
 
 func NewAwsMetadata() (*AwsMetadata, error) {
 	var err error
-	awsMetadata := &AwsMetadata{}
+	awsMetadata := &AwsMetadata{
```

Comment:
Dump question. I know that this is highligthed by the linter, but...
What is the advantage of declaring all the field?
It looks a lot a lot of boilerplate.
Does it improve the code somehow? Or simply readability?

Reply:
Ok! Thank you